### ISSUE TITLE
Need to understand generate component command

### ISSUE THREAD

**Issue by psantos10** [angular/angular-cli#3210]
### OS?
> Windows 10.

### Versions.
> angular-cli: 1.0.0-beta.20-4
> node: 6.9.1
> os: win32 x64

### Repro steps.
> ng g component home

It's not an error. I only want to understand. Why when  I run the previous command it generates the following:

> installing component
>   create src\app\home\home.component.css
>   create src\app\home\home.component.html
>   create src\app\home\home.component.spec.ts
>   create src\app\home\home.component.ts

why not separate, for example, this way:

> installing component
>   create src\app\assets\css\home.component.css
>   create src\app\templates\home.component.html
>   create src\app\tests\home.component.spec.ts
>   create src\app\components\home.component.ts

?

**Comment by grizzm0** [comment]
Angular-cli follows the angular2 [style guide](https://angular.io/styleguide).

**Comment by intellix** [comment]
I think the best place to ask quetions is Gitter :) https://gitter.im/angular/angular-cli
TL;DR - It's better to have everything related to a component together
